tools/python_api_parity/generate_baseline.py

```
@@ -383,45 +383,69 @@ def split_parts(chunk: str) -> list[str]:
+        for match in re.finditer(
+            r"(?m)^\s*(?:pub\s+)?mod\s+([A-Za-z0-9_]+)\s*;", content
+        ):
+            child_path = resolve_module_path(source_path, match.group(1))
+            if child_path is not None:
+                visit(child_path)
+
+    visit(lib_path)
     return sources
 
 
 def parse_rust_surface(repo_root: Path, tier1_rust_symbols: set[str]) -> dict[str, Any]:
```

Ruff (0.15.17)

[warning] Unused function argument: `tier1_rust_symbols`

(ARG001)